cgo/lapack.go

```diff
@@ -102,6 +102,59 @@ type Implementation struct{}
 
 var _ lapack.Float64 = Implementation{}
 
+// Dlacn2 estimates the 1-norm of an n×n matrix A using sequential updates with
+// matrix-vector products provided externally.
+//
+// Dlacn2 is called sequentially. In between calls, x should be overwritten by
+//  A * X if kase == 1
+//  A^T * X if kase == 2
+// all other parameters should be unchanged during sequential calls, and the updated
+// values of est and kase should be used. On the final return (when kase is returned
+// as 0), V = A * W, where est = norm(V) / norm(W).
+//
+// isign, v, and x must all have length n and will panic otherwise. isave is used
+// for temporary storage.
+//
+// Dlacn2 is an internal routine. It is exported for testing purposes.
+func (impl Implementation) Dlacn2(n int, v, x []float64, isgn []int, est float64, kase int, isave *[3]int) (float64, int) {
```

Comment:
I've been thinking about using `[]int` for isave and check its length but I'm not sure it's worth it.